src/relations/database_requires.py

```diff
@@ -158,6 +159,24 @@ def _on_backend_database_created(self, event: DatabaseCreatedEvent) -> None:
         if self.charm.app_peer_data.get(MYSQL_DATABASE_CREATED):
             return
 
+        container = self.charm.unit.get_container(MYSQL_ROUTER_CONTAINER_NAME)
+        # force somehow needed because router bootstrapped for "system" already
+        # (maybe because /etc/mysqlrouter/mysqlrouter.conf exists when router
+        # installed from canonical PPA?)
+        bootstrap_mysql_router_command = f"""
+mysqlrouter
+    --user {MYSQL_USER_NAME}
+    --bootstrap {event.username}:{event.password}@{event.endpoints.split(",")[0]}
+    --conf-base-port 6446
+    --conf-set-option DEFAULT.server_ssl_mode=PREFERRED
+    --conf-set-option http_server.bind_address=127.0.0.1
+    --conf-use-gr-notifications
+    --force
+""".split()
```

Comment:
we need to bootstrap mysqlrouter when the application is scaled to more than 1 unit